NOnion/Network/TorCircuit.fs

```diff
@@ -866,6 +866,9 @@ type TorCircuit
                     | RelayTruncated reason ->
                         let handleTruncated() =
                             match circuitState with
```

Comment:
the 1st line of commit message is a title, therefore doesn't need a dot at the end

Reply:
Removed